src/Router/RoutingServiceProvider.php

```diff
@@ -15,4 +15,40 @@ protected function registerRouter()
             return new CoreRouter($app['events'], $app);
         });
     }
+
+    /**
+     * Register the URL generator service.
+     *
+     * @return void
+     */
+    protected function registerUrlGenerator()
+    {
+        $this->app->singleton('url', function ($app) {
+            $routes = $app['router']->getRoutes();
+
+            // The URL generator needs the route collection that exists on the router.
+            // Keep in mind this is an object, so we're passing by references here
+            // and all the registered routes will be available to the generator.
+            $app->instance('routes', $routes);
+
+            $url = new UrlGenerator(
+                $routes, $app->rebinding(
```

Comment:
If I am not mistaken, the UrlGenerator class is not included in here.

Reply:
@mjauvin You don't have to use a `use` case for a class in the same folder ;)

Reply:
Ah, good to know! :)